packages/toast/src/use-toast.tsx

```diff
@@ -41,11 +41,11 @@ export interface UseToastOptions {
   /**
    * The title of the toast
    */
-  title?: string
+  title?: React.ReactNode
```

Comment:
thinking in some usability, the title can be a string to, like I want to create a toast without jsx element, and the description either the same logic, so, in my mind, we need to accept both types strin | React.ReactNode

Reply:
`ReactNode` includes `ReactChild`, which includes `ReactText` which is a union of `string` and `number` so that should be already covered :)

Reply:
Thinking better, React.ReactNode includes ReactText and this is a union with string and number, so it's ok, Just accept the review. And close this PR